db/queries/assessment_records/_helpers.py

```diff
@@ -61,6 +62,22 @@ def derive_application_values(application_json):
             + f"{application_id}."
         )
         funding_two = 0
+    try:
+        if "COF-R2" in short_ref:  # cof r2
```

Comment:
Shouldn't we be pulling these refs from the mapping above in assessment_mapping_fund_round?

Reply:
ACtually instead of all these if statements, can we just extend the stuff in [config/mappings/assessment_mapping_fund_round.py](https://github.com/communitiesuk/funding-service-design-assessment-store/pull/197/files#diff-5766d9b1f45c9da1bd0580dbbb2e8c58d9c40ca895b9e5ac05a60601bc57200d) to something more like:
`{"NSTFR2": {
    "location": "xyxyxy",
    "asset_type": None,
    "funding_one": "abcabc",
    "funding_two": "werwer" },
"COFR3W1": {
   "location": "oiuoiu",
etc...
}
}`

Then just grab stuff out of there for each specific field we need to populate